Makefile

```diff
@@ -19,17 +19,14 @@ lint-py:
 	@tox -e lint
 
 lint-sh: SHELLCHECK = shellcheck
-lint-sh: BASH_SCRIPTS = $(UA_SCRIPTS)
-lint-sh: DASH_SCRIPTS = $(MOTD_SCRIPTS)
+lint-sh: BASH_SCRIPTS = $(UA_SCRIPTS) $(MOTD_SCRIPTS)
 lint-sh: _lint-sh-command
 
 lint-sh/docker: SHELLCHECK = docker run --rm -it -v $(PWD):/repo koalaman/shellcheck
 lint-sh/docker: BASH_SCRIPTS = $(patsubst %,/repo/%,$(UA_SCRIPTS))
```

Comment:
This should become
```
BASH_SCRIPTS = $(patsubst %,/repo/%,$(UA_SCRIPTS) $(MOTD_SCRIPTS))
```

Reply:
Actually, I think this is the right fix, as the docker run was missing the motd scripts. Do you agree?
```diff
@@ -23,7 +23,7 @@ lint-sh: BASH_SCRIPTS = $(UA_SCRIPTS) $(MOTD_SCRIPTS)
 lint-sh: _lint-sh-command
 
 lint-sh/docker: SHELLCHECK = docker run --rm -it -v $(PWD):/repo koalaman/shellcheck
-lint-sh/docker: BASH_SCRIPTS = $(patsubst %,/repo/%,$(UA_SCRIPTS))
+lint-sh/docker: BASH_SCRIPTS = $(patsubst %,/repo/%,$(UA_SCRIPTS) $(MOTD_SCRIPTS))
 lint-sh/docker: _lint-sh-command
 
 _lint-sh-command:
```